torch/quantization/fx/quantization_patterns.py

```diff
@@ -136,7 +137,12 @@ def get_activation_ctr(
 
 
     @abstractmethod
-    def convert(self, quantizer: QuantizerCls, node: Node, qconfig: QConfigAny, load_arg: Callable,
+    def convert(self,
+                quantizer: QuantizerCls,
+                node: Node,
+                qconfig: QConfigAny,
+                modules: Dict[str, torch.nn.Module],
```

Comment:
in a future PR, could we consider removing this altogether and doing the lookup dynamically?